test/puppetlabs/puppetdb/threadpool_test.clj

```diff
@@ -4,13 +4,11 @@
             [clojure.test :refer :all]
             [puppetlabs.puppetdb.threadpool :refer :all]
             [puppetlabs.puppetdb.testutils :as tu]
-            [puppetlabs.trapperkeeper.testutils.logging :refer [atom-logger
-                                                                with-logging-to-atom
-                                                                with-log-suppressed-unless-notable]]
```

Comment:
As mentioned, I sometimes favor the refers, since in some cases, I find the unprefixed code a bit easier on the eyes.  In any case, while I'm often fine with personal preference on that front, I think I'd keep all the refers in this pr because I think minimizing the number of superfluous changes (line-wise) is the more important consideration (i.e. easier to review, less likely to produce merge conflicts now or in any future merges/cherry-picks, etc.).